src/pages/workspaces/workspace/Data.js

```diff
@@ -211,7 +216,10 @@ class WorkspaceDataContent extends Component {
             return h(DataTypeButton, {
               key: type,
               selected: selectedDataType === type,
-              onClick: () => this.setState({ selectedDataType: type })
+              onClick: () => {
+                saveScroll(0, 0)
```

Comment:
Thinking out loud: This approach requires us to explicitly reset the saved scroll state whenever the component re-mounts. If things change in the future and we forget to do this, it might behave unexpectedly. I wonder if we should have the table call `onScroll` once after mounting, to handle this case automatically. Thoughts?

Reply:
I like this idea--would the change be copying the saveScroll method into table.js and adding saveScroll(0,0) or saveScroll (0,y) at the end of the componentDidMount() for each table?